resources/quality/ultimaker_method/um_method_labs_um-pla-175_0.2mm.inst.cfg

```diff
@@ -55,30 +52,19 @@ speed_wall = =speed_print / 6
 speed_wall_0 = =speed_wall
 speed_wall_x = =speed_wall
 support_angle = 50.0
-support_bottom_distance = =layer_height
-support_bottom_enable = False
-support_fan_enable = True
 support_infill_angles = [45 ]
 support_infill_rate = 12
-support_infill_sparse_thickness = =layer_height
 support_interface_density = 83
 support_interface_height = =layer_height * 4
 support_interface_material_flow = =material_flow * 0.9
-support_interface_offset = 1
-support_interface_wall_count = 0
-support_line_width = line_width * 0.75
+support_line_width = =line_width * 0.75
 support_material_flow = =material_flow * 0.8
 support_offset = 3
 support_roof_density = =support_interface_density
-support_roof_height = =layer_height * 4
 support_roof_line_width = =line_width
-support_roof_material_flow = =support_interface_material_flow
-support_roof_wall_count = 1
+support_top_distance = =support_z_distance
```

Comment:
You seem to be overriding it with the exact same value as the default.

Reply:
this support_top_distance value is set to 0.25 in method_base.def.json, which is not ideal for PLA self support.

Reply:
I changed the support_top_distance for PVA to 0.4mm